frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/select_option_cell/select_option_editor.dart

```diff
@@ -35,13 +35,20 @@ class SelectOptionCellEditor extends StatefulWidget {
 
 class _SelectOptionCellEditorState extends State<SelectOptionCellEditor> {
   late PopoverMutex popoverMutex;
+  final TextfieldTagsController _tagController = TextfieldTagsController();
 
   @override
   void initState() {
     popoverMutex = PopoverMutex();
     super.initState();
```

Comment:
Whilst we're at it, can we switch these two lines?

Reply:
No, there will be no difference in the behavior of the code if you interchange the order of the statements in the initState method. Both versions will achieve the same result.

but yeah for good practices i have switch thease lines in my commit.